web/src/lib/components/shared-components/search-bar/search-text-section.svelte

```diff
@@ -0,0 +1,57 @@
+<script lang="ts">
+  export let fileName: string | undefined;
+  export let context: string | undefined;
+
+  enum TextSearchOptions {
+    Context = 'context',
+    FileName = 'file-name',
+  }
+
+  let selectedOption: TextSearchOptions = TextSearchOptions.Context;
```

Comment:
I think it's better to preselect `File name` when a user previously searched for a filename

```suggestion
  let selectedOption = context ? TextSearchOptions.Context : TextSearchOptions.FileName;
```